composer.json

```diff
@@ -47,7 +47,7 @@
         "kartik-v/yii2-widget-select2": "^2.1",
         "kodus/file-cache": "^1.1",
         "kartik-v/yii2-widget-typeahead": "^1.0",
-        "league/iso3166": "^2.1",
+        "league/iso3166": "^4.0",
```

Comment:
league/iso3166 
- update from 2.1 to 4.0 meant to satisfy requirements for php version 8. 
- 2.1.0 requires php: ^5.6 || ^7.0
- 4.0.0 requires php: ^7.3|^8.0
- it also requires phpunit 9+. same version in use

Reply:
But did you check that our code using it works with version 4?